### DISCUSSION TITLE
Backup User Manager

### DISCUSSION THREAD

**Discussion by 101227936** [beeyev/Mikrotik-RouterOS-automatic-backup-and-update#57] [Ideas]
Besides backup configurations, wish can add backup functionality for user manager database umb

**Comment by beeyev** [comment]
Could you please provide some short proposal, how you see it with command examples?

**Reply by 101227936** [reply]
For example
For backup
user-manager database save name=backup
For restore
user-manager database load name=backup.umb

Can refer on this video
https://m.youtube.com/watch?v=r85YuncwrSk

**Reply by 101227936** [reply]
Wish can make user manager database backup as part of attachment send through email
